packages/desktop-client/src/components/common/View.tsx

```diff
@@ -22,7 +22,7 @@ const View = (props: ViewProps) => {
       {...restProps}
       ref={innerRef}
       style={nativeStyle}
-      className={`view ${css([props.className, props.style])}`}
+      className={`view ${props.className} ${css([props.style])}`}
```

Comment:
```suggestion
      className={`view ${props.className} ${css(props.style)}`}
```
I think the array is no longer needed here? Other than that, looks good! :)